pandora.py

```diff
@@ -165,3 +172,24 @@ def __init__(self):
             'audioFormat': audio_format.value,  # TODO: aacplus and maybe more formats
         })
         return [Track(t) for t in response['tracks']]
+
+    async def rate_track(self, track: Track, track_rating: TrackRating) -> None:
+        """
+        Rate a track.
+
+        :param track: The track to be rated.
+        :param track_rating: The new TrackRating.
+        """
+        if track_rating is TrackRating.NONE:
+            if track.rating is not TrackRating.NONE:
```

Comment:
Wonder if it makes sense to just always send it since it is possible the information we have is outdated?

Reply:
The info we have shouldn't be outdated. You're only *suppose to* be able to listen from one device/app per account at a time.

Reply:
Alright, you'll need to rebase both PRs btw.

Reply:
I know. I've got a lot coming. I'll rebase as you merge them.